modules/virt-about-hot-plugging-virtual-disks.adoc

```diff
@@ -12,4 +12,6 @@ When you _hot unplug_ a virtual disk, you detach a virtual disk from a virtual m
 
 Only data volumes and persistent volume claims (PVCs) can be hot plugged and hot unplugged. You cannot hot plug or hot unplug container disks.
 
-After you hot plug a virtual disk, it remains attached until you detach it, even if you restart the virtual machine.
\ No newline at end of file
+After you hot plug a virtual disk, it remains attached until you detach it, even if you restart the virtual machine.
+
+In the web console, hot-plugged volumes are marked by a label "persistent hotplug" on the disk in the disk list (*VirtualMachine Details* -> *Configuration* -> *Disks* tab). To change a hot-plug volume to a persistent volume, click the Options menu {kebab} and select "Make persistent". The "persistent hotplug" label is removed and change is applied after next reboot.
```

Comment:
```suggestion
In the web console, hot-plugged volumes are marked by a "persistent hotplug" label on the disk in the disk list (*VirtualMachine Details* -> *Configuration* -> *Disks* tab). To change a hot-plug volume to a persistent volume, click the *Options* menu {kebab} and select *Make persistent*. The "persistent hotplug" label is removed and the change is applied after the next reboot.
```

Reply:
From the [SSG](https://redhat-documentation.github.io/supplementary-style-guide/#user-interface-elements): "Bold all graphical user interface elements, including, but not limited to, menu names, menu items, button names, dialog box names, and window names. Bold the item, even if it is not clickable."